### PR TITLE
Write initial positioning lesson

### PR DESCRIPTION
**Pull request by codyloyd**
closes https://github.com/TheOdinProject/top-meta/issues/21 by adding a 'positioning' lesson

### CODE REVIEW

**Review by leosoaivan — COMMENTED** [review]
Small comments, otherwise LGTM

File: html_css/v2/foundations/positioning-and-layout/positioning-and-layout.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+![lines](./imgs/odin-lined.png)
+
+When it comes down to it, laying out a webpage and positioning all of it's elements is basically just deciding how you are going to nest and stack these boxes.
+
+The only real complication here is that there are multiple ways to manipulate the size of these boxes, and the space between them using `margin`, `padding`, and `border`. The assigned articles go into more depth on this concept, but to sum it up briefly: padding increases the space between the edge of a box and the content inside of it, margin increases the space between a box and any others that sit next to it, and border adds space (even if it's only a pixel or two) between the margin and the padding. Be sure to study the diagrams carefully.
+
+![the box model](./imgs/box-model.png)
+
+<div class="lesson-content__panel" markdown="1">
```
Comment (KevinMulhern) [inline html_css/v2/foundations/positioning-and-layout/positioning-and-layout.md:36]: Are we going to have 2 assignments in this lesson? the side nav on lesson pages may have a hissy fit if we do 😆

**Review by KevinMulhern — COMMENTED** [review]
Just one question, everything else looks good 💪

File: html_css/v2/foundations/positioning-and-layout/positioning-and-layout.md (hunk 1 above)
Comment (codyloyd) [reply html_css/v2/foundations/positioning-and-layout/positioning-and-layout.md:36]: I think the side nav just looks for headings.... and there's no heading here.

I could be wrong about that tho

Comment (KevinMulhern) [reply html_css/v2/foundations/positioning-and-layout/positioning-and-layout.md:36]: You're right it does, I still think we should be consistent with 1 assignment box per lesson. But we can come back to that  when we start putting everything together.

**Review by KevinMulhern — APPROVED** [review]
LGTM :shipit:

**Pull request merged**